coord/src/rmr/rmr.c

```diff
@@ -240,7 +239,9 @@ void MR_Init(MRCluster *cl, long long timeoutMS) {
 
   cluster_g = cl;
   timeout_g = timeoutMS;
-  rq_g = RQ_New(8, MAX_CONCURRENT_REQUESTS);
+  // `*50` for following the previous behavior
+  // #define MAX_CONCURRENT_REQUESTS (MR_CONN_POOL_SIZE * 50)
+  rq_g = RQ_New(cl->mgr.nodeConns * 50);
```

Comment:
Is this some kind of rate limiter? can this be avoided? @MeirShpilraien

Reply:
We need to check what happened when we reach this number (I am not sure...)